tests/sentry/grouping/grouping_inputs/cocoa_dispatch_client_callout.json

```diff
@@ -0,0 +1,114 @@
+{
+  "platform": "cocoa",
+  "message": "Foo",
+  "threads": {
+    "values": [
+      {
+        "stacktrace": {
+          "frames": [
+            {
+              "function": "unicorn",
+              "package": "/private/var/containers/Bundle/Application/9B2E1C43-8839-4525-9586-FC58F0534940/Foobar.app/Foobar",
+              "in_app": false,
```

Comment:
The customer has a rule to mark this frame as not to be considered in-app and not considered for grouping, however, we don't take that into consideration for this test. I can remove this frame if it makes anything confusing.